airbnb/src/components/personnel/personnelType.ts

```diff
@@ -0,0 +1,24 @@
+export type GuestInfosType = {
+  title: string;
+  desc: string;
+  state: number;
+  setState: React.Dispatch<React.SetStateAction<number>>;
+}[];
+
+
+type GuestTitle = {
+  title: string
+}
+
+export type GuestAgeProps = GuestTitle & {
+  desc: string;
+};
+
+export type GuestNumProps = GuestTitle & {
+  state: number;
+  setState: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export type GuestItemProps = GuestNumProps & {
+  desc: string;
```

Comment:
왠만하면 풀네임으로 적어주는게 더 명시적이여서 좋을 것 같네요 
```suggestion
  description: string;
```